pkg/audit/annotation.go

```diff
@@ -0,0 +1,37 @@
+package audit
+
+import (
+	"net/http"
+
+	kaudit "k8s.io/apiserver/pkg/audit"
+)
+
+// Decision is a particular type to prevent accidential mis-match of string values.
```

Comment:
This godoc reads strange. What decision does it represent, of what? The sole reason of introducing a type to prevent mismatches is nice but not the driving factor of abstraction.